### ISSUE TITLE
[BUG] Analysis on Gradle application is not raising issues on dependencies

### ISSUE THREAD

**Issue by abrugaro** [konveyor/analyzer-lsp#698]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Konveyor version

0.5

### Priority

Undefined (Default)

### Current Behavior

The analysis does not trigger issues on open source libraries 

### Expected Behavior

The analysis should, at least, trigger an issue on the open source library math3 ([src/main/java/org/apache/commons/math3/distribution/Abstract…apache/commons-math](https://github.com/apache/commons-math/blob/MATH_3_6_1/src/main/java/org/apache/commons/math3/distribution/AbstractRealDistribution.java#L19))

### How Reproducible

Always (Default)

### Steps To Reproduce

1. Create an application using the following Git repository: https://github.com/melix/jmh-gradle-example

2. Start an analysis on it with the following parameters:

Source code + Dependencie
Include Open Source dependencies
No target
Use the following custom rule:
[serializable-alex-rule.txt](https://github.com/user-attachments/files/16897604/serializable-alex-rule.txt)

### Environment

```markdown
- **Analyzer Image SHA** 
- **JDTLS  base image SHA**
- **Provider Settings JSON**
```
